### ISSUE TITLE
Also allow multiple @documentationExample's in javadoc

### ISSUE THREAD

**Issue by vguna** [stoicflame/enunciate#396]
Hi.

Nice feature I found there :).

I like the approach of using non-intrusive javadoc comments for documentation. 👍 

So I started with `@documentationExample`. Coming across a set of enums, I tried to put two examples in the javadoc - without luck. It seems that the annotation pendant _does_ support at least value2.
Are there any plans to support that also for the javadoc part? Or did I miss something?

Currently I return a Set<MyEnum>. As it seems enunciate picks the first value of the enum and creates an array from it. So this leads to an example of:

```json
 "myEnums" : [ "VALUE1", "VALUE1" ],
```
 In theory enunciate could simply pick random values from the enum instead of picking always the first. This way one wouldn't have to specify an explicit example for this. Does this makes sense :)?

**Comment by stoicflame** [comment]
Fixed at 1793a66; random enum values will be selected.

Multiple `@documentationExample` JavaDoc tags can be used for multiple example values; that was already working in 2.3.

**Comment by vguna** [comment]
Thanks for the many fixes! awesome!

But I hate to report, that with 2.3.0 multiple `@documentationExample` 's aren't working. Using `@DocumentationExample` (annotation) with value and value2 works though!

```java
	/**
	 * The field that cause the violation.
	 *
	 * @documentationExample lastname
	 * @documentationExample address.street
	 *
	 * @return the message.
	 */
	// @DocumentationExample(value = "lastname", value2 = "address.street")
	public String getField() {
		return field;
	}
```

**Comment by stoicflame** [comment]
I'm not sure about your use case... that's a single-value property. The second example only comes into play when applied to a property that is a collection or array...

**Comment by vguna** [comment]
I know :). This object is part of a collection (Set). There's an error DTO which has a Set of constraint violation DTOs where one attribute of them is the field above. Isn't this the usecase for the 2nd value :)? If I use the annotation it generates for the error DTO two examples of the constraint violation DTO containg the different examples.

**Comment by stoicflame** [comment]
Okay. Fixed at 7343378.

**Comment by stoicflame** [comment]
[Enunciate 2.4 has been released](https://github.com/stoicflame/enunciate/releases/tag/v2.4.0).
